src/main/java/edu/harvard/iq/dataverse/dataset/DatasetUtil.java

```diff
@@ -568,4 +568,12 @@ public static String getLicenseDescription(DatasetVersion dsv) {
         License license = dsv.getTermsOfUseAndAccess().getLicense();
         return license != null ? license.getShortDescription() : BundleUtil.getStringFromBundle("license.custom.description");
     }
+
+    public static String getLocaleExternalStatus(String status) {
+        String localizedName = BundleUtil.getStringFromBundle(status.toLowerCase().replace(" ", "_"));
```

Comment:
@JayanthyChengan how do you feel about putting these key/value pairs in a new file instead of `Bundle.properties`? This is something @qqmyers and @scolapasta discussed at standup this morning. Maybe a file called `CurationLabels.properties`?

Reply:
@pdurbin - Thanks for your comment. , please review the changes

Reply:
@pdurbin : fixed typo and merged latest develop branch